material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -1118,23 +1118,34 @@ html .md-typeset details.function div.tabbed-set,
 }
 
 /* Feedback widget styling */
-.md-feedback {
-  margin: 1em auto 2em;
-  text-align: center;
-  border: var(--border-width) solid var(--border-purple-bright);
+.feedback-actions-container {
+  margin-top: 2rem;
+  width: calc(100% - 2em);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5em;
+  box-sizing: border-box;
+  border: 1px solid var(--purple-dark-tint-20);
   border-radius: var(--border-radius);
-  padding: 1em 4em;
-  width: fit-content;
+}
+
+.md-feedback {
+  margin-top: 0;
+  margin-bottom: 0;
 }
 
 .md-feedback__title {
+  margin-bottom: 0.75em;
   font-family: var(--md-text-font);
-  font-size: 80%;
+  font-size: 12px;
```

Comment:
With this particular font, this makes the font look really small. I think it should be increased a little bit 
<img width="373" alt="Screenshot 2025-02-03 at 9 25 25 PM" src="https://github.com/user-attachments/assets/53f4a82d-3a03-432e-85fb-acd9bb58f73c" />